src/pcapi/core/offers/api.py

```diff
@@ -321,9 +322,29 @@ def _notify_beneficiaries_upon_stock_edit(stock: Stock):
             # fmt: on
 
 
+def _create_activation_codes(
+    activation_codes: list[str],
+    expiration_datetime: datetime,
+    stock: Stock,
+) -> list[ActivationCode]:
+    result = []
```

Comment:
(détail) 

```suggestion
    created = []
```

pas beaucoup plus concret que `result`, mais un peu plus :)